lib/amazon/ecs.rb

```diff
@@ -347,12 +347,15 @@ def get_hash(element, path='.')
 
         result = element.at_xpath(path)
         if result
-          hash = {}
-          result = result.children
-          result.each do |item|
-            hash[item.name] = item.inner_html
+          if result.children.size == 1 && result.children.first.name == "text"
+            result.children.first.inner_text
```

Comment:
* Interesting patch, but this could easily break existing codes using amazon-ecs. One way to approach this is to introduce and an options hash.
get_hash(element, path, :recursive => true)
* It should always return a hash with the item.name even if the size == 1
* Return value should stick with the existing .inner_html so it is not a breaking change, another option could be introduced to return inner_text
* And please help to update the unit test to cover the new use cases too

Thanks!